app.js

```diff
@@ -1,18 +1,36 @@
 const express = require('express');
 const cors = require('cors');
+const cookieParser = require('cookie-parser'); // FIXME: Keep only if using cookies
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.29.1/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module 'cookie-parser'.